vars/build.groovy

```diff
@@ -17,6 +18,11 @@ def getRepoUrl() {
   return sh(returnStdout: true, script: "git config --get remote.origin.url").trim()
 }
 
+// private
+def getRepoName(repoUrl) {
+  return scm.getUserRemoteConfigs()[0].getUrl().tokenize('/').last().split("\\.")[0]
```

Comment:
The scenario might not happen or be very unlikely but if the work stream the repo was for was delimited by periods or there was is another reason for periods to be included in the repo name, the same value would be returned e.g. `ffc.some-service` and `fcc.another-repo` would both return `ffc`. This [comment](https://stackoverflow.com/questions/45684941/how-to-get-repo-name-in-jenkins-pipeline#comment97915730_45690925) suggests `...split("\\.git")[0]` to ensure the full repo name is retrieved.

Reply:
That makes sense.  Our naming convention shouldn't allow that to happen anyway but it still seems sensible to add it.

Reply:
Added